### ISSUE TITLE
Employer/Group Registration

### ISSUE THREAD

**Issue by ctspukar** [ctspukar/Insurance-Agent-System#42]
When a user register as a group. 
1- Make a login for them along with storing all other information as it is now.
2- Also let them set up security questions so they can answer them in case they forget their password.

3- Once a group is registered, that user can create other user to manage the same group. 
---> The registered user will create username/email and password for new user. 
With that information (username & password), new user can login to the system see the information about the group but also make a link so they can set up security questions /answers.

**Comment by ctspukar** [comment]
@halostream , do you think the description I have is correct. Please confirm . Thanks

**Comment by ctspukar** [comment]
should we change the way employer registration working. For now it says once the account is verified..registered user will get email mentioning username and password to login.

**Comment by saileshpant12** [comment]
@ctspukar 
For now it says once the registrations is verified..registered user will get email mentioning username and password to login. should we change the way employer registration working? I

**Comment by halostream** [comment]
Yes we should if we're having them create their account username and 
password in the registration

**Comment by ctspukar** [comment]
Thanks @halostream  for confirmation. Couple of things we need clarification : 
1- Where exactly in the current process, email is sent to registered employer ?? Is it manual process OR, the application sends the emails after it goes through approval process ? If so where is that approval process ? ---We might need to Remove that section as well .

2- If an employer can create another user to maintain the same group then can that user create another user to again manage the same group ? If not then we need an identifier to know that the registered user is super admin for that group and then all the users super admin creates is just another user with no privileges to create another user.
